### ISSUE TITLE
Custom build

### ISSUE THREAD

**Issue by jarodium** [madrobby/zepto#1361]
Hello 

I would like to suggest that zepto js, could, somehow, provide a builder in the website.
In my use case, i don't require the form and ajax modules, so would like to download from the website a more customized version.

**Comment by tomek-f** [comment]
maybe https://www.npmjs.com/package/zepto-modules ?

**Comment by jarodium** [comment]
Not quite.I would like to avoid nodejs usage. Having a builder that allows me to cherry pick zepto functionality and deliver it as a built js file works best for me.

**Comment by jiaoguanwen** [comment]
same
